### PR TITLE
Merge my Blog and the picture

### PR DESCRIPTION
**Pull request by ShahaamMarzook**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Review by nikit19 — CHANGES_REQUESTED** [review]
Looks good! Just resolve the conflict :+1:

**Review by isuruAb — COMMENTED** [review]
please resolve conflicts.

**Review by ShahaamMarzook — COMMENTED** [review]
Accpt this So,that my task would be apporved

**Comment by ShahaamMarzook** [comment]
Accept my pr please

**Comment by ShahaamMarzook** [comment]
please Merge my pull request

**Comment by ShahaamMarzook** [comment]
@nikit19 please approve this now ASAP
I'm tring to fiish this take for the past two days, willing to do the other tasks.Please approve this as well as my task  https://codein.withgoogle.com/dashboard/task-instances/5733699029041152/

**Comment by ShahaamMarzook** [comment]
Mentors,
   Please approve this and merge this , because I'm trying to pass this for the past 02 days.And help me in completing the task ;
 https://codein.withgoogle.com/dashboard/task-instances/5733699029041152/

**Comment by ShahaamMarzook** [comment]
Mentors,
Please approve this and merge this , because I'm trying to pass this for the past 02 days.And help me in completing the task ;  😄 
https://codein.withgoogle.com/dashboard/task-instances/5733699029041152/?sp-page=2

**Comment by ShahaamMarzook** [comment]
@nikit19 So,have I merged my blog into the main repository.
😮

File: _data/blogs.yml [hunk 1]
```diff
@@ -273,6 +273,11 @@
   img: haider2.jpg
   author_name: Haider Shafaq
 
+- url: https://workingwithfossasiawordpress.wordpress.com/2018/01/06/participation-in-open-source-community/
+  title: Participation in Open source community
+  img: Shahaam.jpeg
```
Comment (nikit19) [inline _data/blogs.yml]: The image that you uploaded and wrote here don't have the same name.
One ends with jpg and the other with jpeg.
Please make it same

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please make this change

**Review by realslimshanky — CHANGES_REQUESTED** [review]
Please squash your commits.

**Comment by ShahaamMarzook** [comment]
I have uploaded the picture in the img/blog As Shahaam.jpg 
Please review my task and accept the pull request as well.   😄 
https://codein.withgoogle.com/dashboard/task-instances/5733699029041152/

With respect,
Shahaam Marzook

**Comment by nikit19** [comment]
But you wrote 'Shahaam.jpeg' in blogs.yml
The name is different,please make it same at both the places

**Review by harshit98 — APPROVED** [review]
Good work!

**Pull request merged**